projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -81,7 +102,9 @@ export class TuiLineClampComponent implements AfterViewInit {
     }
 
     get computedContent(): PolymorpheusContent {
-        return this.overflown ? this.content : '';
+        this.isOverflown$.next(this.overflown);
```

Comment:
May be it is better to move inside `ngDoCheck` ?

Guys, what do you think ?

Reply:
Done